packages/file-search/src/browser/file-search-frontend-module.ts

```diff
@@ -33,4 +34,5 @@ export default new ContainerModule((bind: interfaces.Bind) => {
     );
 
     bind(QuickFileOpenService).toSelf().inSingletonScope();
+    bind(FileQuery).toSelf().inTransientScope();
```

Comment:
Why `inTransientScope`?

Reply:
The intention was to make sure we use a new one every time the container gets an instance, as if the entity was used in other places in addition to the initial server (e.g. by extenders), a new separate instance is provided.
  However it's no longer applicable as the entity does no longer exist.